app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -152,7 +152,7 @@ def execute_planned_action
           run = ::ForemanTasks.dynflow.world.execute(@execution_plan.id)
           if @dynflow_sync_action
             run.wait
-            if run.value.error?
+            if run.value.try(:error?)
```

Comment:
I wonder if we shouldn't check the reason and raise another error when that happens http://ruby-concurrency.github.io/concurrent-ruby/Concurrent/Promises/Future.html#reason-instance_method

When one can get into this state? When the tasks is not running?